src/components/Form/ProductCreateAndEditForm.vue

```diff
@@ -94,16 +86,44 @@
           </v-btn>
         </div>
 
-        <div class="d-flex flex-column mt-4">
-          <p v-if="props.productInfo.resourcesContent?.length > 0">
+        <v-text-field
+          v-if="props.productInfo.resourcesContent?.length > 0"
```

Comment:
I see we use `"props.productInfo.resourcesContent?.length > 0"` in every html tag below. Can we wrap all of them in a single section (maybe div) and then we would write this only once.

Reply:
exported to a component